providers/src/airflow/providers/apache/beam/CHANGELOG.rst

```diff
@@ -542,8 +542,7 @@ Breaking changes
 Breaking changes
 ~~~~~~~~~~~~~~~~
 
-Integration with the ``google`` provider
-````````````````````````````````````````
+Integration with the ``google`` provider.
```

Comment:
This change seems to be unrelated to drill, can you revert this?

Reply:
@jscheffl Reverted this, thanks!

Reply:
Thanks!